orchagent/bfdorch.cpp

```diff
@@ -551,3 +609,113 @@ uint32_t BfdOrch::bfd_src_port(void)
     return (port++);
 }
 
+void BfdOrch::notify_session_state_down(const string& key)
```

Comment:
This is different from the other place, BfdOrch::doTask(NotificationConsumer &consumer). Over there we send these notifications forthe sessions which are already to SAI and have gone down due to loss of connectivity but here we are sending notifications for sessions which have not been created yet. The Vnet Orch expects this status to be posted otherwise it treats the missing status as a error scenario. so we need to provide a dummy down status when the device is in TSA.